components/chart/types/rangearea.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Range Area
+page_title: Chart - Range Area
+description: Overview of the Range Area Chart for Blazor.
```

Comment:
Try to write descriptions that are around 150 symbols and describe the content of the article.